tsconfig.json

```diff
@@ -1,18 +1,27 @@
 {
   "compilerOptions": {
-    "target": "ESNext",
-    "module": "ESNext",
-    "lib": ["ESNext"],
+    "target": "esnext",
+    "module": "commonjs",
```

Comment:
Why commonjs?

Reply:
Because Node ESModule compatibility is bad